package.json

```diff
@@ -26,8 +26,8 @@
   },
   "dependencies": {
     "@actions/core": "^1.2.6",
-    "@carbon/ibmdotcom-react": "^1.18.1",
-    "@carbon/ibmdotcom-styles": "^1.18.1",
+    "@carbon/ibmdotcom-react": "^1.19.0-canary.842286301.0",
+    "@carbon/ibmdotcom-styles": "^1.19.0-canary.842286301.0",
```

Comment:
@guilhermelMoraes there is an automation task that will automatically pull in the next stable release. Is the fix in `v1.19.0`?

Reply:
yeah, it is. In that case, I think we should just close this PR.